internal/xdscache/v3/listener_test.go

```diff
@@ -902,9 +913,11 @@ func TestListenerVisit(t *testing.T) {
 				SocketOptions: envoy_v3.TCPKeepaliveSocketOptions(),
 			}),
 		},
-		"tls-min-protocol-version from config overridden by httpproxy": {
+
+		"tls-protocol-version from config overridden by httpproxy": {
```

Comment:
i think we're missing a case where the maximum is 1.2 but an httpproxy tries to override it to 1.3, the resulting config should still have a max of 1.2